rdt-open-reader/src/test/java/io/ona/rdt/util/UtilsTest.java

```diff
@@ -198,5 +200,9 @@ private void mockStaticClasses() {
         doReturn(FALSE).when(allSharedPreferences).getPreference(IS_IMG_SYNC_ENABLED);
         doReturn(BuildConfig.LOCALE).when(allSharedPreferences).getPreference(Constants.Locale.LOCALE);
         doReturn(allSharedPreferences).when(drishtiContext).allSharedPreferences();
+
+        mockStatic(LangUtils.class);
+        PowerMockito.doNothing().when(LangUtils.class);
+        LangUtils.saveLanguage(ArgumentMatchers.any(Context.class), ArgumentMatchers.anyString());
```

Comment:
This can be merged to one line. i.e. `when(LangUtils.class, "saveLanguage"...)`.

Reply:
fixed.